src/CET.cpp

```diff
@@ -74,7 +74,8 @@ CET::CET()
     , m_vm(m_paths, m_bindings, m_d3d12)
     , m_overlay(m_bindings, m_options, m_persistentState, m_vm)
 {
-    m_vm.Initialize();
+    // Delay our lua hook until later, to ensure that Mod Organizer's VFS is hooked up.
+    //m_vm.Initialize();
```

Comment:
another why is this commented and here, just remove it. that is why version control like git exists.